app/player/elves/elf.py

```diff
@@ -0,0 +1,11 @@
+from app.player.players import Player
+
+
+class Elf(Player):
```

Comment:
Indicate that `Elf` is also an abstract class inheriting it from the `ABC` since it does not implement all abstract methods from the `Player` class.